src/style/core.ts

```diff
@@ -51,7 +51,6 @@ const root = css`
   border: 1px solid var(--rdg-border-color);
   box-sizing: border-box;
   overflow: auto;
-  user-select: none;
```

Comment:
Double clicking on editable cell selects the text first and then the cell goes into edit mode and the editor value is selected again. Not the bet UI but I haven't found a good solution